test/js/ez-toolbar.js

```diff
@@ -1,11 +1,45 @@
 /* global describe, it, assert, fixture */
 describe('ez-toolbar', function() {
-    it('should be defined', function () {
-        const element = fixture('BasicTestFixture');
+    let element;
 
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+    it('should be defined', function () {
         assert.equal(
             window.customElements.get('ez-toolbar'),
             element.constructor
         );
     });
+
+    describe('properties', function () {
+        describe('`visible`', function () {
+            it('should default to false', function () {
+                assert.notOk(element.visible);
+            });
+
+            describe('set', function () {
+                it('should be reflected to an attribute', function () {
+                    const newVisibility = 'Visible';
+
+                    element.visible = newVisibility;
+                    assert.equal(
+                        newVisibility,
+                        element.getAttribute('visible')
+                    );
+                });
+            });
+        });
+    });
+
+    describe('visibility', function () {
+        it('should get the toolbar visible', function () {
+            element.setAttribute('visible', 'visible');
+            assert.equal(window.getComputedStyle(element, null)['display'], 'block', 'toolbar should be displayed');
+        });
+        it('should get the toolbar hidden', function () {
+            element.removeAttribute('visible');
+            assert.equal(window.getComputedStyle(element, null)['display'], 'none', 'toolbar should be displayed');
```

Comment:
aren't you testing that the toolbar is hidden ? :) as I told you yesterday, you can (and should) omit the message when it is the same as the test name.